content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -78,7 +87,7 @@ liveness-6857c8775f-zdv9r        2/2       Running   0           4m
 As stated previously, Istio uses probe rewrite to implement HTTP probes by default. You can disable this
 feature either for specific pods, or globally.
```

Comment:
```suggestion
As stated previously, Istio uses probe rewrite to implement HTTP/TCP probes by default. You can disable this
feature either for specific pods, or globally.
```

I can't leave a suggestion on a line you didn't edit, but please change the heading on L85 to say "the HTTP request/TCP probe" too